src/payload.c

```diff
@@ -506,6 +506,15 @@ size_t getCollectionSize(redfishPayload* payload)
     return (size_t)json_integer_value(count);
 }
 
+size_t getArraySize(redfishPayload *payload)
+{
+    if(!payload)
```

Comment:
Right now, this doesn't compile. I would change it to:
if(!payload || !json_is_array(payload))
{
    return 0;
}
return json_array_size(payload);

Reply:
Thanks, sorry about that.